### DISCUSSION TITLE
Alert panel often not refreshing properly when acking alerts

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#1952] [Unsupported Versions] [answered]
When acknowledging alerts on the Alerts tab, I find quite often that alerts don't get cleared out properly from the view and require a webpage refresh to see them gone. Wondering if anyone else is having this problem with the official 2.3.2 ISO. Both my cpu peaks & ram usage remain consistently below 75%

**Comment by Kennet-B** [comment]
i got the same problem official 2.3.2 ISO

**Comment by jertel** [comment, marked as answer]
I've created an issue for this: https://github.com/Security-Onion-Solutions/securityonion/issues/1956

Thanks for the heads-up.
